python/cudf/cudf/benchmarks/bench_cudf_io.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+
+from cudf.tests import dataset_generator as dg
+
+import pytest
+import cudf
+import numpy as np
+import pandavro as pav
+
+common_dtypes_meta = [
+    [np.dtype("bool"), 0.0, 250],
+    [np.dtype("int8"), 0.1, 481],
+    [np.dtype("int16"), 0.15, 171],
+    [np.dtype("int32"), 0.2, 480],
+    [np.dtype("int64"), 0.3, 177],
+    [np.dtype("uint8"), 0.14, 418],
+    [np.dtype("uint16"), 0.12, 48],
+    [np.dtype("uint32"), 0.17, 177],
```

Comment:
`uint64` is missing.
Are there cudf type traits or type lists in Python? Current implementation is very error-prone.

Reply:
As the Generator is using pyarrow and pandas I had to stick with numpy dtypes.

`uint64` sometimes was creating huge numbers which was failing as system can't handle it.

Reply:
which component is unable to handle it?

Reply:
Python, `OverflowError: Python int too large to convert to C long`

Reply:
@rgsl888prabhu Could you give it another try with latest commits? I made a fix that should now be handling this issue